filebeat/input/file.go

```diff
@@ -75,6 +85,29 @@ func (f *FileEvent) ToMapStr() common.MapStr {
 		"count":      1,
 	}
 
+	if f.jsonDecoder != nil {
+		var jsonObj common.MapStr
+		err := json.Unmarshal([]byte(*f.Text), &jsonObj)
+		if err != nil {
+			logp.Err("Error decoding JSON: %v", err)
+			if f.jsonDecoder.OnUnmarshalError == "add_error_key" {
+				event["json_error"] = fmt.Sprintf("Error decoding JSON: %v", err)
```

Comment:
Question is, what if there are more errors on the same event Do we keep only the latest one? Do we make it an array?